### PR TITLE
Added appropriate action/reaction images from user suggestion.

### PR DESCRIPTION
**Pull request by adv0catus**
https://feathub.com/Mikibot/Miki/+16

### CODE REVIEW

**Comment by velddev** [comment]
No idea what this does. I see a lot of already duplicates.

**Comment by velddev** [comment]
@adv0catus if you have the time, remove the duplicate gifs, and then I'll merge it.

**Comment by adv0catus** [comment]
@velddev This weekend. How do you feel about comments being added beside each imgur link quickly describing it with the intention of reducing/eliminating duplicates?

File: Miki/Modules/ActionsModule.cs [hunk 1]
```diff
@@ -85,7 +85,13 @@ public class ActionsModule
                 "http://i.imgur.com/zG60zPk.gif",
                 "http://i.imgur.com/ct76LIg.gif",
                 "http://i.imgur.com/guBWT22.gif",
-                "http://i.imgur.com/Asnv32U.gif"
+                "http://i.imgur.com/Asnv32U.gif",
+		"http://i.imgur.com/Qv5CAqB.gif",
```
Comment (velddev) [inline Miki/Modules/ActionsModule.cs:89]: Duplicate

File: Miki/Modules/ActionsModule.cs [hunk 2]
```diff
@@ -85,7 +85,13 @@ public class ActionsModule
...
+		"http://i.imgur.com/aHErxah.gif",
+		"http://i.imgur.com/B49G7oQ.gif",
```
Comment (velddev) [inline Miki/Modules/ActionsModule.cs:91]: Duplicate

File: Miki/Modules/ActionsModule.cs [hunk 3]
```diff
@@ -85,7 +85,13 @@ public class ActionsModule
...
+		"http://i.imgur.com/w6cGU77.gif",
```
Comment (velddev) [inline Miki/Modules/ActionsModule.cs:92]: Duplicate

File: Miki/Modules/ActionsModule.cs [hunk 4]
```diff
@@ -85,7 +85,13 @@ public class ActionsModule
...
+		"http://i.imgur.com/X0Ud0Gi.gif",
+		"https://i.imgur.com/fIgeJUr.gif"
```
Comment (velddev) [inline Miki/Modules/ActionsModule.cs:94]: Duplicate

File: Miki/Modules/ActionsModule.cs [hunk 5]
```diff
@@ -222,7 +228,10 @@ public class ActionsModule
                 "http://i.imgur.com/3kerpju.gif",
                 "http://i.imgur.com/uMBRFjX.gif",
                 "http://i.imgur.com/YDJFoBV.gif",
-                "http://i.imgur.com/urC9B1H.gif"
+                "http://i.imgur.com/urC9B1H.gif",
+		"http://i.imgur.com/PaKqsBx.gif",
```
Comment (velddev) [inline Miki/Modules/ActionsModule.cs:232]: Duplicate

File: Miki/Modules/ActionsModule.cs [hunk 6]
```diff
@@ -222,7 +228,10 @@ public class ActionsModule
...
+		"http://i.imgur.com/u4iUyeH.gif",
+		"http://i.imgur.com/ta1u312.gif"
```
Comment (velddev) [inline Miki/Modules/ActionsModule.cs:234]: Duplicate

File: Miki/Modules/ActionsModule.cs [hunk 7]
```diff
@@ -271,7 +280,8 @@ public class ActionsModule
                 "http://i.imgur.com/XLjH6zQ.gif",
                 "http://i.imgur.com/W7arBvy.gif",
                 "http://i.imgur.com/W9htMol.gif",
-                "http://i.imgur.com/IVOBC8p.gif"
+                "http://i.imgur.com/IVOBC8p.gif",
+		"http://i.imgur.com/eYf50FP.gif"
```
Comment (velddev) [inline Miki/Modules/ActionsModule.cs:284]: Odd aspect ration

File: Miki/Modules/ActionsModule.cs [hunk 8]
```diff
@@ -359,6 +364,9 @@ public class ActionsModule
                 "http://i.imgur.com/iekwz4h.gif",
                 "http://i.imgur.com/q7AmR8n.gif",
                 "http://i.imgur.com/pDohPrm.gif",
+		"http://i.imgur.com/eib0EBw.gif",
+		"http://i.imgur.com/XGpq4S7.gif",
```
Comment (velddev) [inline Miki/Modules/ActionsModule.cs:368]: Duplicate

File: Miki/Modules/ActionsModule.cs [hunk 9]
```diff
@@ -359,6 +364,9 @@ public class ActionsModule
...
+		"http://i.imgur.com/uvXG6uU.gif"
```
Comment (velddev) [inline Miki/Modules/ActionsModule.cs:369]: Duplicate

File: Miki/Modules/ReactionsModule.cs [hunk 10]
```diff
@@ -251,7 +251,8 @@ public class ReactionsModule
 				"http://i.imgur.com/pkT7JFw.gif",
 				"http://i.imgur.com/Tx15hPX.gif",
 				"http://i.imgur.com/YANiZ2a.gif",
-				"http://i.imgur.com/31WnXZ7.gif"
+				"http://i.imgur.com/31WnXZ7.gif",
+				"http://i.imgur.com/u1wxWx9.gif"
```
Comment (velddev) [inline Miki/Modules/ReactionsModule.cs:255]: Odd aspect ratio

**Review by velddev — CHANGES_REQUESTED** [review]

**Pull request closed without merging**